pkg/restapi/handler.go

```diff
@@ -27,6 +27,9 @@ func Start() error {
 	r.HandleFunc("/backup/{id}", deleteBackup).Methods(http.MethodDelete)
 	r.HandleFunc("/next-batch", backupScheduler).Methods(http.MethodGet)
 
+	r.HandleFunc("/restore", createRestore).Methods(http.MethodPost)
+	r.HandleFunc("/restore/{id}", getRestore).Methods(http.MethodGet)
+
 	log.Println("Starting API...")
```

Comment:
should this be log.info

Reply:
No. This is using the `golang` log module.